modules/distribution/src/repository/conf/identity/service-providers/default.xml

```diff
@@ -0,0 +1,67 @@
+<!--
+  ~ Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+  ~
+  ~ WSO2 Inc. licenses this file to you under the Apache License,
+  ~ Version 2.0 (the "License"); you may not use this file except
+  ~ in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied. See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+
+<ServiceProvider>
+    <ApplicationID>1</ApplicationID>
+    <ApplicationName>default</ApplicationName>
+    <Description>Default Service Provider</Description>
+    <InboundAuthenticationConfig>
+        <InboundAuthenticationRequestConfigs>
+            <InboundAuthenticationRequestConfig>
+                <InboundAuthKey>default</InboundAuthKey>
+                <InboundAuthType></InboundAuthType>
+                <Properties></Properties>
+            </InboundAuthenticationRequestConfig>
+        </InboundAuthenticationRequestConfigs>
+    </InboundAuthenticationConfig>
+    <LocalAndOutBoundAuthenticationConfig>
+        <AuthenticationSteps>
+            <AuthenticationStep>
+                <StepOrder>1</StepOrder>
+                <LocalAuthenticatorConfigs>
+                    <LocalAuthenticatorConfig>
+                        <Name>BasicAuthenticator</Name>
+                        <DisplayName>basicauth</DisplayName>
+                        <IsEnabled>true</IsEnabled>
+                    </LocalAuthenticatorConfig>
+                </LocalAuthenticatorConfigs>
+                <!-- FederatedIdentityProviders>
+                 <IdentityProvider>
+                       <IdentityProviderName>facebook</IdentityProviderName>
+                       <IsEnabled>true</IsEnabled>
+                                     <DefaultAuthenticatorConfig>
+                                             <FederatedAuthenticatorConfig>
+                                                     <Name>FacebookAuthenticator</Name>
+                                                     <IsEnabled>true</IsEnabled>
+                                             </FederatedAuthenticatorConfig>
+                                     </DefaultAuthenticatorConfig>
+                 </IdentityProvider>
```

Comment:
Already fixed in commit https://github.com/wso2-incubator/product-iot-server/commit/febdc36cba8b94c3ebcaba27947893ebb2865513 as per the PrabathA's comment.